smartmeter/smartmeter.ino

```diff
@@ -22,6 +22,10 @@
 #define YELLOW_LED D7
 #define GREEN_LED D4
 #define RELAY D2
+#define CURRENT_SENSOR A0
+#define BUTTON 14 // D5
+
+volatile bool buttonState = false;
```

Comment:
I don't think this variable is necessary, can't you do everything with the Data?

Reply:
Done

Reply:
This variable is not used.

Reply:
Removed